### PR TITLE
[db_manager] Use connector.quoteString in query builder

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Fix https://hub.qgis.org/issues/14796

This apply on query builder dialog, where we adapt `Column's values` quoting depending on datatype and database provider.

### CODE REVIEW

File: python/plugins/db_manager/dlg_query_builder.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def list_values(self):
 
     def query_item(self, index):
         queryWord = index.data()
-        queryWord = ' "%s"' % queryWord
+        queryWord = ' %s' % self.db.connector.quoteString(queryWord)
         if queryWord != '':
```
Comment (sbrunner) [inline python/plugins/db_manager/dlg_query_builder.py]: Always true (queryWord can't be en empty string...)

**Comment by sbrunner** [comment]
This will not works for the numbers...

**Comment by arnaud-morvan** [comment]
This should handle most of standard types now.

**Comment by sbrunner** [comment]
Looks relay better :-)

**Comment by arnaud-morvan** [comment]
@slarosa I think this should me merged before the freeze, does this looks good for you.

**Comment by slarosa** [comment]
@arnaud-morvan yes looks fine to me. Sorry for the delay, too many things to think in this period. I got a chance to review this change and I have merged in 8e5b28f with a small modification to the commit text by adding the word ```Fix #14796``` in order to close the issue automatically in the bug tracker. Thanks.

**Pull request closed without merging**